.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
```

Comment:
To check this in we'd have to discuss its addition first on a project level. Please propose that separately and take the file out of this specific PR.

Reply:
+1.